modules/nodes-pods-user-namespaces-configuring.adoc

```diff
@@ -0,0 +1,200 @@
+====
++
+After you save the changes, new machine configs are created, the machine config pools are updated, and scheduling on each node is disabled while the change is being applied.
+
+* After the nodes return to a `READY` state, enable the crun container runtime to use with new containers on the worker nodes:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: ContainerRuntimeConfig
+metadata:
+ name: enable-crun-worker
+spec:
+ machineConfigPoolSelector:
+   matchLabels:
+     pools.operator.machineconfiguration.openshift.io/worker: "" <1>
+ containerRuntimeConfig:
+   defaultRuntime: crun <2>
+----
+<1> Specifies the machine config pool label. 
+<2> Specifies the container runtime to deploy to new containers. The default value is `runc`.
+
+.Procedure
+
+. Edit the default user ID (UID) and group ID (GID) range of the {product-title} namespace where your pod is deployed by running the following command:
++
+[source,terminal]
+----
+$ oc edit ns/<namespace_name>
+----
++
+.Example namespace
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/description: ""
+    openshift.io/display-name: ""
+    openshift.io/requester: system:admin
+    openshift.io/sa.scc.mcs: s0:c27,c24
+    openshift.io/sa.scc.supplemental-groups: 1024/10000 <1>
+    openshift.io/sa.scc.uid-range: 1024/10000 <2>
+# ...
+name: userns
+# ...
+----
+<1> Edit the default GID to match the value you specified in the pod spec. The range for a Linux user namespace must be lower than 65,535. The default is `1000000000/10000`.
+<2> Edit the default UID to match the value you specified in the pod spec. The range for a Linux user namespace must be lower than 65,535. The default is `1000000000/10000`.
++
+[NOTE]
+====
+The range 1024/10000 means 10,000 values starting with ID 1024, so it specifies the range of IDs from 1024 to 11,023.
+====
+
+. Enable the use of Linux user namespaces by creating a pod configured to run with a `restricted` profile and with the `hostUsers` parameter set to `false`:
+
+.. Create a YAML file similar to the following:
++
+.Example pod specification
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: userns-pod
+
+# ...
+
+spec:
+  containers:
+  - name: userns-container
+    image: registry.access.redhat.com/ubi9
+    command: ["sleep", "1000"]
+    securityContext:
+      capabilities:
+        drop: ["ALL"]
+      allowPrivilegeEscalation: false <1>
+      runAsNonRoot: true <2>
+      seccompProfile:
+        type: RuntimeDefault
+      runAsUser: 1024 <3>
+      runAsGroup: 1024 <4>
+  hostUsers: false <5>
+
+# ...
+----
+<1> Specifies that a pod cannot request privilege escalation. This is required for the `restricted` security context constraints (SCC).
```

Comment:
After checking scc `restricted-v2` and `restricted`,  "allowPrivilegeEscalation: false" belong to `restricted-v2`. 
It should be: This is required for the `restricted-v2` security context constraints (SCC).